ocs_ci/ocs/rados_utils.py

```diff
@@ -323,3 +325,70 @@ def corrupt_pg(osd_deployment, pool_name, pool_object):
     )
     osd_pod.exec_cmd_on_pod(original_osd_cmd + " " + original_osd_args)
     ct_pod.exec_ceph_cmd(f"ceph pg deep-scrub {pgid}")
+
+
+def inject_corrupted_dups_into_pg_via_cot(
+    osd_deployment, pgid, injected_dups_file_name_prefix="text"
+):
+    """
+    Inject corrupted dups into a pg via COT
+
+    Args:
+        osd_deployment (OCS): List of OSD deployment OCS instances
```

Comment:
It will be more clear if we change it to `osd_deployments`

Reply:
Done